docs/reference/modules/shard-ops.asciidoc

```diff
@@ -0,0 +1,69 @@
+[[shard-allocation-relocation-recovery]]
+=== Shard allocation, relocation, and recovery
+
+Each <<documents-indices,index>> in Elasticsearch is divided into one or more <<scalability,shards>>, each of which acts as a copy of the index and can be replicated across multiple <<modules-node,nodes>> to protect against hardware failures.
+
+include::{es-ref-dir}/primaries-replicas-desc.asciidoc[]
+
+Over the course of normal operation, Elasticsearch allocates shards to nodes, relocates shards across nodes to balance the cluster, and recovers shards to initialize new copies. In this topic, you'll learn how these operations work and how you can control them.
```

Comment:
Is it worth mentioning here that shard movement can happen not only for balancing but also to satisfy allocation constraint (or decider) such as data tier?